src/Layer/Launchpad/Launchpad.lua

```diff
@@ -118,9 +118,30 @@ function Launchpad:send(channel, number, value)
     if (not self.midi_out or not self.midi_out.is_open) then
         return
     end
-    local message = {channel, number, value}
-    -- print(("Launchpad : send MIDI %X %X %X"):format(message[1], message[2], message[3]))
-    self.midi_out:send(message)
+	if (channel == 0x90 and value > 127) then
```

Comment:
Ok, I will create a more complex color set (according to the features of the launchpad pro) and map it to the old launchpad, this way people can enjoy the new features of the new launchpad. But not in this PR. This time I just want to make it work.